test/jaxp/javax/xml/jaxp/unittest/transform/SurrogateTest1.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<root>
+    <tag1>𠮟</tag1>
+</root>
```

Comment:
Add a new line at the end of the file (and other newly created files too).

Reply:
I added a new line at the end of every new file.